data/json/items/armor/boots.json

```diff
@@ -1503,7 +1537,28 @@
         "encumbrance": 15
       }
     ],
-    "melee_damage": { "bash": 1 }
+    "melee_damage": { "bash": 1 },
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "generic_knee_high_boots",
+        "name": { "str": "pair of knee-high boots", "str_pl": "pairs of knee-high boots" },
+        "description": "Very long leather boots that cover the lower legs, they are cumbersome to wear.",
+        "weight": 10
+      },
+      {
+        "id": "jackboots",
+        "name": { "str": "pair of jackboots", "str_pl": "pairs of jackboots" },
+        "description": "A pair of sturdy, over-the-knee, laceless boots fitted with a rubberized sole: these imposing combat boots provide good traction despite their encumbering nature.  Most commonly associated with totalitarian regimes due to their use by Nazi soldiers, this style of boot is warm, protective, and produces a deeply satisfying sound while marching.",
+        "weight": 10
+      },
+      {
+        "id": "chelsea_boots",
+        "name": { "str": "pair of chelsea boots", "str_pl": "pairs of chelsea boots" },
+        "description": "A pair of knee-length, laceless boots formed from calfskin with an elastic side panel that’s designed to stretch away to facilitate simple dawning and doffing, these stylish stompers are a time-old gift from Victorian-age England.  Intended for you to fashionably saunter down roadways rather than sprint for your life, this style of boot became a style staple within London during the 1960s, where they were synonymous with the mod croud.",
```

Comment:
```suggestion
        "description": "A pair of knee-length, laceless boots formed from calfskin with an elastic side panel that’s designed to stretch away to facilitate simple dawning and doffing, these stylish stompers are a time-old gift from Victorian-age England.  Intended for you to fashionably saunter down roadways rather than sprint for your life, this style of boot became a style staple within London during the 1960s, where they were synonymous with the mod crowd.",
```